test/integration/targets/azure_rm_webapp/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/azure
+posix/ci/cloud/group2/azure
```

Comment:
The aliases were just recently changed, see https://github.com/ansible/ansible/blob/devel/test/integration/targets/azure_rm_loadbalancer/aliases for the new format.

Reply:
resolved